hdtv/plugins/calInterface.py

```diff
@@ -188,18 +201,32 @@
         try:
             self.spectra.dict[spectrumID].effCal.TF1.Draw()
         except AttributeError:
-            raise hdtv.cmdline.HDTVCommandError("No efficiency for spectrum ID %d set" % spectrumID)
-
-    def Fit(self, spectrumIDs, filename, nuclides, coefficients, sigma,
-            show_graph=False, fit_panel=False, show_table=False, source=None):
+            raise hdtv.cmdline.HDTVCommandError(
+                "No efficiency for spectrum ID %d set" % spectrumID
+            )
+
+    def Fit(
```

Comment:
Codacy found an issue: [EffCalIf.Fit is too complex (16) (MC0001)](https://app.codacy.com/manual/janmayer/hdtv/pullRequest?prid=6293112)